packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -631,17 +629,6 @@ export class MLSService extends TypedEventEmitter<Events> {
     return this.coreCryptoClient.wipeConversation(groupIdBytes);
   }
 
-  public async handleEvent(params: Omit<EventHandlerParams, 'mlsService'>): EventHandlerResult {
-    return handleBackendEvent({...params, mlsService: this}, async groupId => {
-      const conversationExists = await this.conversationExists(groupId);
-      if (!conversationExists) {
-        return;
-      }
-      const newEpoch = await this.getEpoch(groupId);
-      this.emit('newEpoch', {groupId, epoch: newEpoch});
-    });
```

Comment:
`newEpoch` event is now emmited in `handleMLSMessageAdd` directly, no need to pass it as callback function